packages/renderer/src/lib/welcome/WelcomePage.svelte

```diff
@@ -0,0 +1,83 @@
+<script lang="ts">
+import bgImage from './background.png';
+import DesktopIcon from '../images/DesktopIcon.svelte';
+import Fa from 'svelte-fa/src/fa.svelte';
+import { faCircle } from '@fortawesome/free-solid-svg-icons';
+import { onMount } from 'svelte';
+import { WelcomeUtils } from './welcome-utils';
+
+let showWelcome;
+
+$: showWelcome = false;
+
+onMount(async () => {
+  const welcomeUtils = new WelcomeUtils();
+  const ver = await welcomeUtils.getVersion();
+  if (ver === undefined) {
+    welcomeUtils.updateVersion('initial');
+    showWelcome = true;
+  }
+});
+</script>
+
+{#if showWelcome}
+  <div
+    class="flex flex-col flex-auto fixed top-10 left-0 right-0 bottom-0 bg-zinc-700 overflow-y-auto z-50"
+    style="background-image: url({bgImage}); background-position: 50% -175%; background-repeat: no-repeat; background-size: 100% 75%">
```

Comment:
https://tailwindcss.com/docs/background-repeat

bg-no-repeat

Reply:
See second commit. I moved to bg-no-repeat but left the dynamic url for now because using Svelte {bgImage} felt cleaner than Tailwind url, but this is a judgement call so let me know if you disagree.